docs/guides/Guide-Intro.md

```diff
@@ -66,7 +68,7 @@ class HomeScreen extends React.Component {
   }
 }
 
-export default const SimpleApp = StackNavigator({
+export const SimpleApp = StackNavigator({
```

Comment:
We had `export default` here so that it would work with `create-react-native-app`. We followed the pattern that [React Native](https://facebook.github.io/react-native/docs/tutorial.html#hello-world) is using. What was the issue that was being caused by `export default`?

Reply:
leaving `default` gives me back an `unexpected token` error (even via CRNA).

That said, I'm having troubles making it work with CRNA at all. I'll probably update my PR once I've managed to make it work the CRNA way.